### PR TITLE
vw_golf_mk4: translate signals to english as comments episode 2

### PR DESCRIPTION
**Pull request by rusefillc**
A bit more of the same

### CODE REVIEW

**Comment by sshane** [comment]
I think the convention so far is not to include spaces in between messages

**Comment by rusefillc** [comment]
@sshane huh? https://github.com/commaai/opendbc/pull/653/files was merged a few hours ago

**Comment by sshane** [comment]
Right, we should revert that. We should do all DBCs or none, and it'd probably be its own PR if it's a standard.

**Comment by rusefillc** [comment]
I do not follow since spaces were used prior to my merge but I hope you guys will figure out and document your convention if you have one.

https://github.com/commaai/opendbc/blob/5ce12a2558e5657b62936d6346b7b05baecf6646/vw_golf_mk4.dbc#L1437

**Comment by jyoung8607** [comment]
I've seen several DBCs written by commercial tools with an extra newline between blocks of signal comments. I'm not married to either way of doing it, but neither way seems like a violation to me, and I like it if there's going to be a lot of comments.

**Comment by rusefillc** [comment]
So when @sshane says "spaces" he means "empty lines between content lines"? I am not a native speaker that was confusing.

**Comment by jyoung8607** [comment]
While any translation work is interesting, the most helpful would be translations and descriptions of the signals used by openpilot (commaai/openpilot#24768). We use a variety of signals within the following messages:

* Bremse_1
* Bremse_3
* Lenkhilfe_3
* Lenkwinkel_1
* Motor_3
* Airbag_1
* Bremse_5
* GRA_Neu
* Kombi_1
* Motor_2
* Lenkhilfe_2
* Gate_Komf_1
* ACC_GRA_Anziege
* SWA_1
* LDW_1
* HCA_1

Within those messages, the signals actively used in openpilot carstate and carcontroller are the most helpful.

**Comment by jyoung8607** [comment]
BTW, if you are still actively working on a given PR (as seems the case here) use the "convert to Draft" function to let other potential contributors and reviewers know it's not yet ready to be reviewed and accepted.

File: vw_golf_mk4.dbc [hunk 1]
```diff
@@ -1433,19 +1433,51 @@ BO_ 2000 Diagnose_1: 8 XXX
  SG_ DI1_KM_Stand_alt : 62|1@1+ (1,0) [0|1] "" XXX
  SG_ DI1_Zeit_alt : 63|1@1+ (1,0) [0|1] "" XXX
 
+CM_ SG_ 80 Checksumme_Airbag_1 "Checksum Airbag_1"
```
Comment (sshane) [inline vw_golf_mk4.dbc:1436]: Can you make sure add semicolons and remove the newlines?

Comment (rusefillc) [reply vw_golf_mk4.dbc:1436]: @sshane do you mean https://github.com/commaai/opendbc/pull/654/commits/984a9ab3668812d494b3ad9e8c02ae47e8f8815a when you request new lines to be removed? Seems to be a negative value but it's your call.

@sshane please clarify what semicolors are you referring to. I am a bit confused if there is some pattern/style/convention I am missing or if that's a random subjective preference?

Comment (jyoung8607) [reply vw_golf_mk4.dbc:1436]: The [DBC specification](http://mcu.so/Microcontroller/Automotive/DBC_File_Format_Documentation.pdf) calls for a trailing semicolon on the comment lines:

```suggestion
CM_ SG_ 80 Checksumme_Airbag_1 "Checksum Airbag_1";
```

**Comment by rusefillc** [comment]
ooops, my bad. fixed.

File: vw_golf_mk4.dbc (hunk 1 above)
Comment (adeebshihadeh) [reply vw_golf_mk4.dbc]: @rusefillc we generally don't have strict stylistic guidelines. it's up to the author to make their code tasteful, and we'll review as needed. in this case, I think you were right with the newlines. @sshane why did you want them removed?

**Pull request merged**